mypy/plugins/attrs.py

```diff
@@ -212,7 +212,7 @@ def _determine_eq_order(ctx: 'mypy.plugin.ClassDefContext') -> bool:
     order = _get_decorator_optional_bool_argument(ctx, 'order')
 
     if cmp is not None and any((eq is not None, order is not None)):
-        ctx.api.fail("Don't mix `cmp` with `eq' and `order`", ctx.reason)
+        ctx.api.fail('Don\'t mix "%s" with "%s" and "%s"' % (cmp, eq, order), ctx.reason)
```

Comment:
This change makes the error message worse. We shouldn't be interpolating the values for these arguments here.
```suggestion
        ctx.api.fail('Don\'t mix "cmp" with "eq" and "order"', ctx.reason)
```

Reply:
@hauntsaninja Below code looks good? 
```python
ctx.api.fail('Don\'t mix "{}" with "{}" and "{}"'.format(cmp, eq, order), ctx.reason)
```

Reply:
or should i change it to 
```python
ctx.api.fail('Don\'t mix {} with {} and {}'.format(cmp, eq, order), ctx.reason)
```